README.md

```diff
@@ -4,7 +4,7 @@ This plugin will guarantee that when running the repl, you are always up to date
 
 ## Usage
 
-Put `[komcrad/lein-autoreload "0.2.0"]` into the `:plugins` vector of your
+Put `[wsbu/lein-autoreload "0.2.0"]` into the `:plugins` vector of your
```

Comment:
This is referring to the project's groupId, not the GitHub organization name. It should be `net.redlion`, and it should match the groupId in `project.clj`

Reply:
Fixed with changes to project.clj and readme